tests/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "hubspot-cli-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "run-tests.js",
+  "scripts": {
+    "test": "node run-tests",
+    "lint": "eslint .",
+    "prettier": "prettier --write './*.{js,json}'"
+  },
```

Comment:
Linting and prettier should be handled by the top-level package. We also should consider incorporating with `lerna` so that stuff "just works" when working locally.